src/how_to/nominate.md

```diff
@@ -27,6 +27,28 @@ If your comment requires more than 5-10 minutes of reading and discussion to und
 
 We try to be prompt, but sometimes we are not. Othertimes, we discuss the issue, but fail to leave the follow-up comment, because we're only human. Please feel free to raise the topic on Zulip or reach out to a lang team member.
 
+### "Drag" factor and "easy decision"
+
+The language team is experimenting with a **"drag" factor** based system for sorting agenda items. Items are quantized into five drag factor levels, `P-lang-drag-{0,1,2,3,4}`. It is dubbed a *drag factor* to avoid conflating it with priority or "worthiness" of an agenda item, but is instead a combination of factors. Items with *lower* drag factors get ranked *higher* on the agenda. Drag factor 0 agenda items come first while drag factor 4 agenda items come last during a lang meeting.
+
+> **Example**
+>
+> An agenda item may be (1) very import, (2) very hard, and (3) non-urgent, so it may receive a high drag factor.
```

Comment:
is "very import" a typo? ~~or doge speak?~~

but according to the description below if the phrase means "very important", the item should have a *low* drag factor not a *high* one.

Reply:
Ugh this is worded confusingly I agree, I tried to transcribe

> It's a "drag" factor, so lower values get ranked higher on the agenda. It's called "drag" because we're not expressing some evaluation about the inherent worthiness or priority of the issue overall. It's a combination of factors. Something could be really important, but also really hard (e.g. we already know members disagree on it) and non-urgent, and so it'll get a high drag factor.

but my wording here does not faithfully represent that TC said.

Reply:
Much better, thanks!